Dockerfile

```diff
@@ -258,6 +258,15 @@ COPY ./contrib/dockerd-rootless.sh /build
 
 FROM djs55/vpnkit:${VPNKIT_VERSION} AS vpnkit
 
+FROM base AS dockerd-rootless-setuptool
+ENV INSTALL_BINARY_NAME=dockerd_rootless_setuptool
+COPY . /go/src/github.com/docker/docker
```

Comment:
avoid this huge copy with `--mount` please

Reply:
fixed